test_mysteryword.py

```diff
@@ -0,0 +1,121 @@
+import mysteryword as mw
+
+
+def test_word_getter_got_something():
+    word = mw.word_getter()
+    assert word is not None
+
+
+def test_word_getter_got_string():
+    word = mw.word_getter()
+    assert type(word) == str
```

Comment:
Tests of `type` are generally not great. I care more about what I got and what it can do.